RELEASE.yaml

```diff
@@ -0,0 +1,21 @@
+description: |
```

Comment:
@julenpardo please note that this is a public repo -- please make sure it's ok to expose such information.

also pinging @yakov-g for review

Reply:
Yes right, I double checked with @carsongee before